alex_px-moscow.md

```diff
@@ -3,5 +3,7 @@
 ### Restaurants
 
 ### Local Attractions
+- Red Square
+- Historical Museum
 
 ### Remote Attractions
```

Comment:
:wave:

Any special events going on this month?

When's the next national holiday?